Makefile

```diff
@@ -226,6 +226,13 @@ urn:
 	@printf "⚱\n\n";
 	@$(MAKE) run;
 
+docker-build:
+	docker build -t wp-calypso .
+
+docker-run:
+	docker run -it --name wp-calypso --rm -p 80:3000 -e NODE_ENV='wpcalypso' -e CALYPSO_ENV='wpcalypso' wp-calypso
```

Comment:
it may be worth indicating somehow that this is running the `wpcalypso`

even better would be a sensible default indicated on the command line and then the option to run in different environments